main/mainworker.cpp

```diff
@@ -2055,7 +2058,7 @@ void MainWorker::ProcessRXMessage(const CDomoticzHardwareBase *pHardware, const
 
 	const_cast<CDomoticzHardwareBase *>(pHardware)->SetHeartbeatReceived();
 
-	char szDate[100]; szDate[0] = 0;
+	char szDate[100];
```

Comment:
Have a look on github and compare current against your changes
My changes are needed to not log a date/time when this is disabled via the command prompt (for example for syslog users)